docker/sonar/sonarqube-compose.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Google LLC
```

Comment:
Can we also have an README.md file containing details on how to run the sonar locally.

Reply:
we can bring the changes here and deprecate the old one.

Reply:
Done.